content/en/docs/how-tos/migrating-template-jobs-to-multistage.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Migrating CI Jobs from Templates to Multi-stage Tests"
+date: 2020-10-30T21:49:06+02:00
+draft: false
+---
+
+OpenShift CI offers two mechanisms for setting up end-to-end tests: older 
+template-based tests and newer [multi-stage test workflows](/docs/architecture/step-registry/).
+Template-based tests cause problems for both DPTP and OpenShift CI users:
+ - they need intensive support
+ - backing implementation is buggy and notoriously difficult to maintain
+ - jobs using templates are hard to extend, modify, and troubleshoot
+
+These concerns were addressed in designing the multi-stage workflows, which
+supersede template-based tests. DPTP wants to migrate all existing template-based
+jobs to multi-stage workflows in the medium term.
+
+## Migrating Jobs Generated from `ci-operator` Configurations
+
+Most template-based jobs are generated from `ci-operator` configuration stanzas.
+Migrating these jobs is easier and can be done almost mechanically. Soon, DPTP
+will migrate all existing `ci-operator` configurations to multi-stage workflows
+automatically.
```

Comment:
maybe this is too obvious but mention that we expect this to cause no service disruption